pipenv/utils.py

```diff
@@ -843,6 +843,15 @@ def is_installable_file(path):
         path = urlparse(path['file']).path if 'file' in path else path['path']
     if not isinstance(path, six.string_types) or path == '*':
         return False
+    # If the string starts with a valid specifier operator, test if it is a valid
```

Comment:
this seems like this should be in a seperate pull request, i have no idea what this is doing.

Reply:
it was required to pass appveyor tests in this one.